saleor/product/models.py

```diff
@@ -376,8 +376,8 @@ class Meta:
     def __str__(self):
         return self.name
 
-    def get_formfield_name(self):
-        return slugify('attribute-%s' % self.slug)
+    def get_formfield_name(self, allow_unicode=True):
```

Comment:
Do we know of any cases where `get_formfield_name` would be passed this argument?